src/interfaces/modular/GaussianProcess.i

```diff
@@ -16,12 +16,14 @@
 %rename(InferenceMethod) CInferenceMethod;
 %rename(ExactInferenceMethod) CExactInferenceMethod;
 %rename(LaplacianInferenceBase) CLaplacianInferenceBase;
+%rename(FITCInferenceBase) CFITCInferenceBase;
```

Comment:
@lisitsyn can you comment on exposing abstract base classes via the modular interfaces?